packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -701,6 +705,7 @@ export class Viewer {
    */
   public async handleWebSocketClose(): Promise<void> {
     if (this.isStreamStarted) {
+      console.log('handle ws closed');
```

Comment:
do you want to leave this for continued visibility?

Reply:
Nope, removed.